### PR TITLE
Merged all the projects under the Cargo workspace

### PR DESCRIPTION
**Pull request by kvark**
This will speed up the builds a little. Also added CI testing for the sample/replay binaries.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/510)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[workspace]
+members = ["webrender", "webrender_traits", "sample", "replay"]
```
Comment (larsbergstrom) [inline Cargo.toml]: Total nit: typically, we would put each of these on their own line and sort them. e.g.:
```
[workspace]
members = [
  "replay",
  "sample",
  "webrender",
  "webrender_traits",
]
```

**Comment by kvark** [comment]
Thanks @larsbergstrom for your points! Should be good now.

**Pull request merged**